### ISSUE TITLE
Metadata inconsistencies when uploading files through web without proper EXIF information

### ISSUE THREAD

**Issue by danieldietzler** [immich-app/immich#14191]
### The bug

If the client has any filesystem metadata information about the time of a file it'll pass that to the server. However, those aren't actually persisted to a sidecar file (but written to the database), *even if* the file does not actually have any proper EXIF information. This results in Immich showing metadata the original file does not have, which is confusing. 

Thus, that time should ideally be written to EXIF in a sidecar file to ensure the metadata between database and files on disk matches.

### The OS that Immich Server is running on

N/A

### Version of Immich Server

v1.120.2

### Version of Immich Mobile App

N/A

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

N/A

**Comment by C-Otto** [comment]
Writing the data to another kind of storage, you suggested the XMP file, does not change anything about the confusion. I don't think Immich's internal storage format should be "confusion free" to users, as long as the information provided via the web frontend and app is sane. As a user, you're free to change the date of an asset, and you shouldn't be too concerned with how that's done under the hood.

I'm closing this issue as I don't think there's anything we should do.

**Comment by mmomjian** [comment]
@C-Otto I think the issue here is that all other methods of setting the asset time do (or should) create an XMP file. So it’s rather confusing that this method sets the time but doesn’t generate a matching XMP.

**Comment by C-Otto** [comment]
It's under the hood and not part of the public API (or is it?), so users shouldn't worry about confusing internals.

**Comment by mmomjian** [comment]
I respectfully disagree. We regularly tell people that metadata edits are saved in the sidecar, and I think this is part of the immich design philosophy

**Comment by danieldietzler** [comment]
Independent of what's happening "under the hood", the database and file system state should never get out of state. This is a case of that. Why it gets out of sync in the first place, *that's* an implementation detail the user shouldn't worry about, yes. Even if you don't care about anything else, an inconsistent file system at the very least provides some sort of vendor lock-in, which is bad.

**Comment by jrasm91** [comment]
We now write/update the utimes on the file with the information that's provided in the upload request. If that information is available in the client, and it is sent, the server will sync it to the filesystem.
